tests/test_interface.rb

```diff
@@ -40,7 +40,8 @@ class TestInterface < CiscoTestCase
   DEFAULT_IF_IP_PROXY_ARP = false
   DEFAULT_IF_IP_REDIRECTS = true
   DEFAULT_IF_VRF = ''
-  IF_DESCRIPTION_SIZE = 243 # SIZE = VSH Max 255 - "description " keyword
+  IF_DESCRIPTION_VALID_SIZE = 68 # SIZE = VSH Max 80 - "description " keyword
+  IF_DESCRIPTION_INVALID_SIZE = 255
```

Comment:
So even though it says 'max size allowed is 80', it actually means 'max size allowed is 63'? That seems odd.

If you can just add some comments explaining why these numbers are selected, I'd be happy - it's just that with the current lack of comments, it's not at all obvious why INVALID_SIZE != VALID_SIZE + 1.